From 7dec14c032a405b3fdbae9270645e2405222529b Mon Sep 17 00:00:00 2001
From: Suman Anna <s-anna@ti.com>
Date: Fri, 3 Feb 2017 10:50:27 -0600
Subject: [PATCH 191/205] ARM: dts: AM33xx: Add label property to PRU nodes

Add a 'label' property to each of the PRU nodes within the PRU-ICSS
subsystem. Each of the PRU nodes on AM33xx SoCs have been updated
with either a "pru0" or "pru1" label value.

Signed-off-by: Suman Anna <s-anna@ti.com>
(cherry picked from commit 6e8d98131478737c7f17d6e3a8f40d5395aaeb64)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm/boot/dts/am33xx.dtsi |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/am33xx.dtsi b/arch/arm/boot/dts/am33xx.dtsi
index 87c33dd..c01fc03 100644
--- a/arch/arm/boot/dts/am33xx.dtsi
+++ b/arch/arm/boot/dts/am33xx.dtsi
@@ -984,6 +984,7 @@
 					      <0x4a322000 0x400>,
 					      <0x4a322400 0x100>;
 					reg-names = "iram", "control", "debug";
+					label = "pru0";
 					interrupt-parent = <&pruss_intc>;
 					interrupts = <16>, <17>;
 					interrupt-names = "vring", "kick";
@@ -996,6 +997,7 @@
 					      <0x4a324000 0x400>,
 					      <0x4a324400 0x100>;
 					reg-names = "iram", "control", "debug";
+					label = "pru1";
 					interrupt-parent = <&pruss_intc>;
 					interrupts = <18>, <19>;
 					interrupt-names = "vring", "kick";
-- 
1.7.9.5

